src/current/v25.4/ltree.md

```diff
@@ -0,0 +1,272 @@
+---
+title: LTREE
+summary: The LTREE data type stores hierarchical tree-like structures as label paths.
+toc: true
+docs_area: reference.sql
+---
+
+The `LTREE` [data type]({% link {{ page.version.version }}/data-types.md %}) stores hierarchical tree-like structures as a *label path*, which is a sequence of dot-separated *labels*. Labels represent positions in a tree hierarchy. `LTREE` is useful for efficiently querying and managing hierarchical data without using recursive joins.
+
+## Syntax
+
+Each label in an `LTREE` label path represents a level in the hierarchy, beginning from the root (`Animals` in the following example):
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT 'Animals.Mammals.Carnivora'::LTREE;
+~~~
+
+~~~
+            ltree
+-----------------------------
+  Animals.Mammals.Carnivora
+~~~
+
+Each label in the path must contain only alphanumeric characters (`A`-`Z`, `a`-`z`, `0`-`9`), underscores (`_`), and hyphens (`-`).
+
+The following are valid `LTREE` values:
+
+- `'Animals'`
+- `'Products.Electronics.Laptops.Gaming'`
+- `'project_a.phase-1.task_001'`
+- `''` (empty path)
+
+## Size
+
+The size of an `LTREE` value is variable and equals the total number of characters in all labels plus the dot separators. The maximum label length is 1,000 characters, and the maximum number of labels in a path is 65,535.
```

Comment:
That makes the max LTREE ~65MB. For STRING values we recommend keeping them under 64 kB:

https://www.cockroachlabs.com/docs/stable/string#size

> The size of a STRING value is variable, but it's recommended to keep values under 64 kilobytes to ensure performance. Above that threshold, [write amplification](https://www.cockroachlabs.com/docs/v25.3/architecture/storage-layer#write-amplification) and other considerations may cause significant performance degradation.

Since an LTREE is stored like a STRING, I think we should make a similar recommendation here.